presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestQueryConfigurationOverridesConfig.java

```diff
@@ -48,14 +49,16 @@ public void testExplicitPropertyMappings()
                 .put("password-override", "_password")
                 .put("session-properties-override-strategy", "SUBSTITUTE")
                 .put("session-properties-override", "{\"key\": \"value\"}")
+                .put("session-properties-removal", "key2")
```

Comment:
nit - pass in multiple values to test the splitting logic - `key3,key4`

Reply:
✅